.github/workflows/ci.yml

```diff
@@ -623,27 +630,27 @@ jobs:
           python3 "$SCRIPT_DIR/../local-network/authorize-minters.py"
 
           echo "Waiting for quiet after authorizing minters..."
-          PRE_MINT_BLOCK_INDEX=$("$BIN_DIR/mc-consensus-tool" wait-for-quiet --beyond-block=$PRE_AUTH_BLOCK_INDEX)
+          PRE_MINT_BLOCK_INDEX=$("$BIN_DIR/mc-consensus-tool" wait-for-quiet --beyond-block="$PRE_AUTH_BLOCK_INDEX")
           echo "Done waiting, PRE_MINT_BLOCK_INDEX=${PRE_MINT_BLOCK_INDEX}"
 
           # Mint 1 million token1's to the first 4 accounts
           echo "Minting"
           for ACCOUNT_NUM in $(seq 0 4); do
               "$BIN_DIR/mc-consensus-mint-client" \
-                  "generate-and-submit-mint-tx" \
+                  generate-and-submit-mint-tx \
                   --node insecure-mc://localhost:3200/ \
                   --signing-key "$BIN_DIR/mc-local-network/minting-keys/minter1" \
-                  --recipient $(cat keys/account_keys_${ACCOUNT_NUM}.b58pub) \
+                  --recipient "$(cat "keys/account_keys_${ACCOUNT_NUM}.b58pub")" \
```

Comment:
This looks like it might have issues with quoting and escaping?

Reply:
This does work, its weird, but because of the `()` the internally quoted considered separately.